lib/matplotlib/tests/test_backend_pdf.py

```diff
@@ -132,3 +129,34 @@ def test_grayscale_alpha():
     ax.imshow(dd, interpolation='none', cmap='gray_r')
     ax.set_xticks([])
     ax.set_yticks([])
+
+
+@cleanup
+def test_pdfpages_accept_pep_519():
```

Comment:
The test_pdfpages_accept_pep_519 and test_savefig_accept_pathlib? I'm not sure it's possible to skip part of a test and have that appear in nose/pytest.